texts/formatted_harry_potter.txt

```diff
@@ -0,0 +1,1679 @@
+;	"as" cnjadv
+	"as" pr
+"<the>"
+	"the" det def sp
+"<Ministry>"
+	"ministry" n sg
+"<of>"
+	"of" pr
+"<Magic>"
+;	"magic" adj
+	"magic" n sg
+"<and>"
+	"and" cnjcoo
+"<subjugate>"
+	"subjugate" vblex inf
+;	"subjugate" vblex pres
+;	"subjugate" vblex imp
+"<all>"
+;	"all" adv
+;	"all" predet sp
+;	"all" det qnt pl
+	"all" prn qnt mf sp
+"<wizards>"
+	"wizard" n pl
+"<and>"
+	"and" cnjcoo
+"<Muggles>"
+	"*Muggles"
+"<(>"
+	"(" lpar
+"<non>"
+	"non" adv
+"<->"
+	"-" guio
+"<magical>"
+	"magical" adj
+"<people>"
+;	"people" n sg
+	"person" n pl
+"<)>"
+	")" rpar
+"<.>"
+	"." sent
+"<Since>"
+;	"since" adv
+;	"since" cnjsub
+	"since" pr
+"<the>"
+	"the" det def sp
+"<release>"
+	"release" n sg
+;	"release" vblex inf
+;	"release" vblex pres
+;	"release" vblex imp
+"<of>"
+	"of" pr
+"<the>"
+	"the" det def sp
+"<first>"
+;	"first" adv
+	"first" adj ord
+"<novel>"
+;	"novel" adj
+	"novel" n sg
+"<,>"
+	"," cm
+"<Harry>"
+	"Harry" np ant m sg
+"<Potter>"
+;	"potter" n sg
+	"Potter" np cog sg
+"<and>"
+	"and" cnjcoo
+"<the>"
+	"the" det def sp
+"<Philosopher>"
+	"philosopher" n sg
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<Stone>"
+	"stone" n sg
+;	"stone" vblex inf
+;	"stone" vblex pres
+;	"stone" vblex imp
+;	"Stone" np cog sg
+;	"Stone" np ant m sg
+"<,>"
+	"," cm
+"<on>"
+;	"on" adv
+	"on" pr
+"<26>"
+	"26" num
+"<June>"
+	"June" n sg
+"<1997>"
+	"1997" num
+"<,>"
+	"," cm
+"<the>"
+	"the" det def sp
+"<books>"
+	"book" n pl
+;	"book" vblex pres p3 sg
+"<have>"
+;	"have" vbhaver inf
+	"have" vbhaver pres
+;	"have" vblex inf
+;	"have" vblex pres
+;	"have" vblex imp
+"<found>"
+;	"found" adj
+;	"find" vblex past
+	"find" vblex pp
+;	"found" vblex inf
+;	"found" vblex pres
+;	"found" vblex imp
+"<immense>"
+	"immense" adj
+"<popularity>"
+	"popularity" n sg
+"<,>"
+	"," cm
+"<critical>"
+	"critical" adj
+"<acclaim>"
+	"acclaim" vblex inf
+;	"acclaim" vblex pres
+;	"acclaim" vblex imp
+"<and>"
+	"and" cnjcoo
+"<commercial>"
+	"commercial" adj
+;	"commercial" n sg
+"<success>"
+	"success" n sg
+"<worldwide>"
+;	"worldwide" adj
+	"worldwide" adv
+"<.>"
+	"." sent
+"<They>"
+	"prpers" prn subj p3 mf pl
+"<have>"
+;	"have" vbhaver inf
+	"have" vbhaver pres
+;	"have" vblex inf
+;	"have" vblex pres
+;	"have" vblex imp
+"<attracted>"
+	"attract" vblex pp
+;	"attract" vblex past
+"<a>"
+	"a" det ind sg
+"<wide>"
+	"wide" adj sint
+"<adult>"
+	"adult" adj
+;	"adult" n sg
+"<audience>"
+	"audience" n sg
+"<as well as>"
+	"as well as" cnjcoo
+"<younger>"
+	"young" adj sint comp
+"<readers>"
+	"reader" n pl
+"<and>"
+	"and" cnjcoo
+"<are>"
+	"be" vbser pres
+"<often>"
+	"often" adv
+"<considered>"
+	"consider" vblex pp
+;	"consider" vblex past
+"<cornerstones>"
+	"cornerstone" n pl
+"<of>"
+	"of" pr
+"<modern>"
+	"modern" adj
+"<young>"
+	"young" adj sint
+"<adult>"
+	"adult" adj
+;	"adult" n sg
+"<literature>"
+	"literature" n sg
+"<.>"
+	"." sent
+"<As>"
+;	"as" adv
+;	"as" preadv
+;	"as" cnjsub
+;	"as" cnjadv
+	"as" pr
```

Comment:
the tagging guidelines don't address this, but 2 prepositions in a row seems a little odd to me - might want to ask about this on IRC